### PR TITLE
The actual information has been added to the page "Contact us".

### PR DESCRIPTION
**Pull request by iambrozjak**
dev
https://github.com/ita-social-projects/ZeroWaste/issues/690

Code reviewers
 @obniavko 
 @loqimean

Summary of issue
The page "Contact us "displays outdated information. The link does not work .

Summary of change

The actual information has been added to the page "Contact us". The link to the official website is now working. 
contact -us

### CODE REVIEW

**Review by obniavko — APPROVED** [review]

**Review by mdotvasylenchuk — APPROVED** [review]

File: app/views/messages/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <div class="text">
+              <%= t('.title') %>
+            </div>
+            <%= form.text_field :title, class:'form-control', placeholder:t('.title_hint'), minlength:"5", required: true %>
+          </div>
+          <div class="email">
+            <div class="text">
+              <%= t('.email') %>
+            </div>
+            <%= form.email_field :email, class:'form-control', placeholder:'example@example.com', required: true %>
```
Comment (mdotvasylenchuk) [inline app/views/messages/new.html.erb:19]:
```suggestion
            <%= form.email_field :email, class:'form-control', placeholder: 'example@example.com', required: true %>
```

File: app/views/messages/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <h2 class="text-center p-2">
+        <%= t('.contact_us_header') %>
+      </h2>
+      <div class="contact-form">
+        <%= form_with model: @message do |form| %>
+          <div class="title">
+            <div class="text">
+              <%= t('.title') %>
+            </div>
+            <%= form.text_field :title, class:'form-control', placeholder:t('.title_hint'), minlength:"5", required: true %>
```
Comment (mdotvasylenchuk) [inline app/views/messages/new.html.erb:13]:
```suggestion
            <%= form.text_field :title, class:'form-control', placeholder: t('.title_hint'), minlength: "5", required: true %>
```

File: app/views/messages/new.html.erb
Comment (mdotvasylenchuk) [inline app/views/messages/new.html.erb:1]: додай спейси в хешах

Comment (iambrozjak) [reply app/views/messages/new.html.erb:1]: done

File: app/views/messages/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        <% end %>
+      </div>
+    </div>
+  </div>
+  <br>
+  <div class="contact-info card-block p-3 m-auto">
+    <div class="adress">
+      <span class="fas fa-globe"></span>
+      <span class="text">
+        <%= link_to 'zerowastelviv.org.ua', 'http://zerowastelviv.org.ua', target: '_blank' %>
```
Comment (loqimean) [inline app/views/messages/new.html.erb:39]: я б це взагалі в майбутньому додав в сайт-сеттінги, це окремою таскою, дадай на борду пліз

File: app/views/messages/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            <div class="text">
+              <%= t('.email') %>
+            </div>
+            <%= form.email_field :email, class:'form-control', placeholder: 'example@example.com', required: true %>
+          </div>
+          <div class="msg">
+            <div class="text">
+              <%= t('.message') %>
+            </div>
+            <%= form.text_area :message, class:'form-control', placeholder:(t('.message_hint')), size:"25x2", minlength:"20", required: true %>
```
Comment (loqimean) [inline app/views/messages/new.html.erb:25]:
```suggestion
            <%= form.text_area :message, class: 'form-control', placeholder: (t('.message_hint')), size: "25x2", minlength:"20", required: true %>
```

Comment (loqimean) [reply app/views/messages/new.html.erb:25]: от чому нема спейсів..?

**Review by loqimean — APPROVED** [review]

**Pull request merged**